app/build.gradle

```diff
@@ -7,6 +7,7 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'realm-android'
 
+def STATIC_API_KEY = '"'+StaticTokenKey+'"' ?: '"Define your static api key"'
```

Comment:
This is not using environment variable.

Reply:
@iamareebjamal Sir, Shall I use System.getenv() ?

Reply:
yes